src/payment/strategies/masterpass/masterpass-payment-strategy.ts

```diff
@@ -26,21 +26,18 @@ export default class MasterpassPaymentStrategy implements PaymentStrategy {
 
     initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
         const { methodId } = options;
-        const state = this._store.getState();
-        const storeConfig = state.config.getStoreConfig();
+        const { language: localeLanguage } = navigator;
```

Comment:
You should read from the locale provided by the storefront rather than the browser. This is because going forward, it is a value that can be configured by merchants. The value should already be passed into `CheckoutService` so you should be able to pass it along to the various strategies for Masterpass (https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/payment/create-payment-strategy-registry.ts#L76).